tests/base/symmetric_tensor_39.cc

```diff
@@ -44,6 +44,9 @@ int main()
           counter += 1.0;
         }
 
+  // without this we get wrong results with clang in release mode
+  std::cout << '\0';
```

Comment:
I ask because I think that `'\0'` prints a character that has no screen representation. But it's still in the output file. That's undesirable. Would it be enough to print a space, or an empty string?